js/pages/connections.jsx

```diff
@@ -17,109 +30,156 @@ module.exports = React.createClass({
           return {
             Address: peer.substr(0, slashIndex),
             ID: peer.substr(slashIndex+1)
-          }
+          };
         });
 
         peers = peers.sort(function(a, b) {
-          return a.ID > b.ID ? 1 : -1
-        })
+          return a.ID > b.ID ? 1 : -1;
+        });
         peers.map(function(peer) {
-          peer.ipfs = t.props.ipfs
-          peer.location = { formatted: '' }
+          peer.ipfs = t.props.ipfs;
+          peer.location = { formatted: '' };
 
-          var location = t.state.locations[peer.ID]
+          var location = t.state.locations[peer.ID];
           if(!location) {
-            t.state.locations[peer.ID] = {}
+            t.state.locations[peer.ID] = {};
             t.props.ipfs.id(peer.ID, function(err, id) {
-              if(err) return console.error(err)
+              if(err) return console.error(err);
 
               getLocation(id.Addresses, function(err, res) {
-                if(err) return console.error(err)
+                if(err) return console.error(err);
 
-                res = res || {}
-                peer.location = res
-                t.state.locations[peer.ID] = res
+                res = res || {};
+                peer.location = res;
+                t.state.locations[peer.ID] = res;
                 t.setState({
                   peers: peers,
                   locations: t.state.locations,
                   nonce: t.state.nonce++
-                })
-              })
+                });
+              });
             });
           }
-        })
-      })
-    }
```

Comment:
i prefer no semicolons, its ok if some sneak in, but i'm not for changing the style to have semicolons because i dont want to make this codebase a ground for semilcolon wars.